src/main/java/duke/commands/EventCommand.java

```diff
@@ -0,0 +1,20 @@
+package duke.commands;
+import duke.Ui;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+
+/**
+ * Command to add an event to a task list.
+ */
+public class EventCommand extends Command {
+    private String requestContent;
```

Comment:
Seems like you prefer a line space between variables and methods, consider putting one here?